samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -7,20 +7,54 @@ Bluetooth: Mesh light switch
    :local:
    :depth: 2
 
-The Bluetooth mesh light switch sample demonstrates how to set up a basic mesh client model application, and control LEDs with the Bluetooth mesh using the :ref:`bt_mesh_onoff_readme`.
+The :ref:`ug_bt_mesh` light switch sample can be used to change the state of light sources on other devices within the same mesh network.
+It also demonstrates how to use Bluetooth mesh models by using the Generic OnOff model in an application.
+
+Use the light switch sample with the :ref:`bluetooth_mesh_light` sample to demonstrate its function in a basic Bluetooth mesh network.
```

Comment:
```suggestion
Use the light switch sample with the :ref:`bluetooth_mesh_light` sample to demonstrate its function in a Bluetooth mesh network.
```

Otherwise, a reader would wonder what is 'basic' Bluetooth mesh network.